pkgs/tools/filesystems/e2fsprogs/default.nix

```diff
@@ -1,5 +1,21 @@
-{ stdenv, buildPackages, fetchurl, fetchpatch, pkgconfig, libuuid, gettext, texinfo }:
+{ stdenv, buildPackages, fetchurl, fetchpatch, pkgconfig, libuuid, gettext, texinfo, lib }:
 
+let
+  # Remove some failing tests
+  # https://github.com/NixOS/nixpkgs/issues/65471
+  disabledTests = [
+    "m_image_mmp"
+    "m_mmp"
+    "m_mmp_bad_csum"
+    "m_mmp_bad_magic"
+    "t_mmp_1on"
+    "t_mmp_2off"
+  ] ++
+    # Remove a test that fails on 32bit ARMv7 architectures
+    # https://github.com/NixOS/nixpkgs/issues/72473
+    (lib.optional (stdenv.hostPlatform.parsed.cpu.name == "armv7l") "m_hugefile");
```

Comment:
Tbh, it feels wrong to disable tests just because of Raspbian. If it works on NixOS, I'd rather keep the test enabled. Besides, this commit doesn't technically fit the title of the pull request. I'll remove this commit.